### PR TITLE
Update TBG_macros.cfg

### PR DESCRIPTION
**Pull request by Heikman**
add chargeConservation plugin

~~wait till #790 is merged~~ (not necessary)

### CODE REVIEW

File: doc/TBG_macros.cfg [hunk 1]
```diff
@@ -196,6 +196,8 @@ TBG_liveViewYX="--live_e.period 1 --live_e.slicePoint 0.5 --live_e.ip 10.0.2.254
 TBG_liveViewYZ="--live_e.period 1 --live_e.slicePoint 0.5 --live_e.ip 10.0.2.254 \
                 --live_e.port 2021 --live_e.axis yz"
 
+# print the maximum charge deviation between particles and div E
+TBG_chargeConservation="--chargeConservation_period 100"
```
Comment (ax3l) [inline doc/TBG_macros.cfg]: pls use `chargeConservation.period` in the plugin (as all other plugins should do)

**Comment by PrometheusPi** [comment]
@Heikman Thanks. Would it not be better to add this to pull request #790?

**Comment by ax3l** [comment]
I thought that too but I do not mind.

**Comment by PrometheusPi** [comment]
This should **not go to** `master`. I will thus close it.

**Comment by ax3l** [comment]
@Heikman please update the `TBG_macros.cfg` in #790

**Pull request closed without merging**